requirements/base.txt

```diff
@@ -20,7 +20,7 @@ watchdog==2.1.2
 rich~=13.4.2
 pyopenssl~=23.2.0
 # Pin to <4.18 to until SAM-T no longer uses RefResolver
-jsonschema<4.18
+jsonschema<4.20
```

Comment:
Does this comment still hold?

Reply:
good catch, need to check

Reply:
SAM-T depends on `jsonschema<5,>=3.2` see [here](https://github.com/aws/serverless-application-model/blob/00245b65ca324a418ebe713359350fab829b964a/requirements/base.txt#L2)

Reply:
It looks like `cfn-lint` [requires](https://github.com/aws-cloudformation/cfn-lint/blob/bf7b7afa3a67d1f58f3003a2d627f14d97141492/setup.py#L49) `jsonschema>=3.0,<4.18`, so even if we did bump it in the requirements file, the reproducible text files will likely still be pinned to `4.17.x`

Reply:
We still might be able to bump this on our side since `cfn-lint` sets to bounds to something lower anyways